dev-docs/workflows/logcollection.md

```diff
@@ -0,0 +1,53 @@
+# Logcollection
+
+One can deploy [Filebeat](https://www.elastic.co/guide/en/beats/filebeat/current/index.html) and [Logstash](https://www.elastic.co/guide/en/logstash/current/index.html) to enable collection of logs to [OpenSearch](https://search-e2e-logs-y46renozy42lcojbvrt3qq7csm.eu-central-1.es.amazonaws.com/_dashboards/app/home#/), which allows for agreggation and easy inspection of said logs.
+The logcollection functionality can be deployed to both [debug](./debug-cluster.md) and non-debug clusters.
+
+## Deployment in Debug Clusters
+
+In debug clusters, logcollection functionality should be deployed automatically through the debug daemon `debugd`, which runs *before* the bootstrapper and can therefore, contrary to non-debug clusters, also collect logs of the bootstrapper.
+
+## Deployment in Non-Debug Clusters
+
+In non-debug clusters, logcollection functionality needs to be explicitly deployed as a Kubernetes Deployment through Helm. To do that, a few steps need to be followed:
+
+1. Template the deployment configuration through the `loco` CLI.
+
+    ```bash
+    bazel run //hack/logcollector template -- \
+        --dir="." --username="<OPENSEARCH_USERNAME>" \
+        --password="<OPENSEARCH_PW>" \
+        --info key=value \
+        --info key2=value2
+        ...
+    ```
+
+    This will place the templated configuration in the current directory.
+    OpenSearch user credentials can be created by any admin in OpenSearch. Logging in with your company CSP accounts should grant you sufficient permissions to [create a user](https://opensearch.org/docs/latest/security/access-control/users-roles/#create-users) and [grant him the required `all_access` role](https://opensearch.org/docs/latest/security/access-control/users-roles/#map-users-to-roles).
+    One can add additional key-value pairs to the configuration by appending `--info key=value` to the command. These key-value pairs will be attached to the log entries and can be used to filter them in OpenSearch.
+2. Deploy Logstash
+
+    ```bash
+    cd Logstash
```

Comment:
```suggestion
    # assuming at repo root
    cd hack/logcollector/internal/templates/logstash/
```

Reply:
This would head into the templates directory, which is not correct. We want to get into the directory where the templated things get placed. If the command from step 1 has been executed (`--dir="."`), `cd logstash` is correct

Reply:
yeah just `logstash` now its working

Reply:
```suggestion
    cd logstash
```